temporary-table.md

```diff
@@ -0,0 +1,332 @@
+
+- The table definition of a local temporary table is not persistent. A local temporary table is visible only to the session in which the table is created, and other sessions cannot access the table.
+- You can create local temporary tables with the same name in different sessions, and each session reads only from and write only to the local temporary table created in the session.
+- The data of a local temporary table is visible to all transactions in the session.
+- After a session ends, the local temporary table created in the session is automatically deleted.
+- A local temporary table can have the same name with an ordinary table. At this time, in the DDL and DML statements, the ordinary table is hidden until the local temporary table is deleted.
+
+You can create a local temporary table using the `CREATE TEMPORARY TABLE` statement, and delete a local temporary table using the `DROP TABLE` or `DROP TEMPORARY TABLE` statement.
+
+Different from MySQL, the local temporary tables in TiDB are all external tables, and you cannot using SQL statements to create internal temporary tables.
+
+### Usage examples of local temporary tables
+
+> **Note:**
+>
+> - Before you use the temporary table in TiDB, pay attention to the [compatibility restrictions with other TiDB features](#compatibility-restrictions-with-other-tidb-features) and the [compatibility with MySQL temporary tables](#compatibility-with-mysql-temporary-tables).
+> - If you have created temporary tables on a cluster earlier than v5.3.0, these tables are ordinary tables, and treated as ordinary tables after the cluster is upgraded to v5.3.0 or later.
+
+Assume that there is an ordinary table `users`:
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE users (
+    id BIGINT,
+    name VARCHAR(100),
+    PRIMARY KEY(id)
+);
+```
+
+In session A, creating a local temporary table `users` does not conflict with the ordinary table `users`. When session A accesses the `users` table, it accesses the local temporary table `users`.
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TEMPORARY TABLE users (
+    id BIGINT,
+    name VARCHAR(100),
+    city VARCHAR(50),
+    PRIMARY KEY(id)
+);
+```
+
+```
+Query OK, 0 rows affected (0.01 sec)
+```
+
+If you insert data into `users`, data is inserted to the local temporary table `users` in session A.
+
+{{< copyable "sql" >}}
+
+```sql
+INSERT INTO users(id, name, city) VALUES(1001, 'Davis', 'LosAngeles');
+```
+
+```
+Query OK, 1 row affected (0.00 sec)
+```
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT * FROM users;
+```
+
+```
++------+-------+------------+
+| id   | name  | city       |
++------+-------+------------+
+| 1001 | Davis | LosAngeles |
++------+-------+------------+
+1 row in set (0.00 sec)
+```
+
+In session B, creating a local temporary table `users` does not conflict with the ordinary table `users`. When session B accesses the `users` table, it accesses the local temporary table `users`.
```

Comment:
```suggestion
In session B, creating a local temporary table `users` does not conflict with the ordinary table `users` or the local temporary table `users` in session A. When session B accesses the `users` table, it accesses the local temporary table `users`.
```

Reply:
Do we need to add this to the Chinese version?